lambdas/indexer/app.py

```diff
@@ -137,3 +147,14 @@ def aggregate_retry(event: chalice.app.SQSEvent):
                     batch_size=1)
 def contribute_retry(event: chalice.app.SQSEvent):
     app.index_controller.contribute(event, retry=True)
+
+
+if config.enable_log_forwarding:
+    @app.on_s3_event(bucket=aws.qualified_bucket_name(config.logs_term),
+                     events=['s3:ObjectCreated:*'],
+                     # https://aws.github.io/chalice/api.html#Chalice.on_s3_event
+                     # This a literal string prefix, not a glob, so we can't use "*"
+                     # to forward from multiple deployments.
```

Comment:
We have to forward the ALB logs from each private deployment as well as those from GitLab. It would be OK to have each deployment have it's own forwarder and the main deployment additionally forward the logs from the GitLab ALB. It would also be OK to have the main deployment forward everything. The former feels more appropriate to me.

Reply:
Decided at PL to go with the latter instead